src/api/storage-pools.tsx

```diff
@@ -31,11 +31,9 @@ export const fetchStoragePool = (
   });
 };
 
-export const fetchStoragePools = (
-  project: string,
-): Promise<LxdStoragePool[]> => {
+export const fetchStoragePools = (): Promise<LxdStoragePool[]> => {
```

Comment:
wouldn't `fetchStoragePool` above also have the same issue? I.e. we should remove the project query param

Reply:
Good point, actually all the pool API functions should have the project parameter removed. Amended the changes just now.